services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/client/gRPC/GoalStateClientImpl.java

```diff
@@ -244,10 +302,18 @@ public void onCompleted() {
         };
 
         StreamObserver<Goalstate.GoalStateV2> requestObserver = asyncStub.pushGoalStatesStream(responseObserver);
+        long requestObserverEstablished = System.currentTimeMillis();
+        logger.log(Level.FINE, "[doSendGoalState] Established RequestObserver, elapsed Time after stub established in milli seconds: " + (requestObserverEstablished - stub_established));
```

Comment:
This logging is quite extensive. Do we have an easy to turn it off when it is not needed..

Reply:
I can change these logs' level to FINE, and if we specify the log level to INFO in the `application.properties` then these logs should not show.